packages/bindgen/spec.yml

```diff
@@ -680,8 +681,8 @@ classes:
 
   ObjectStore:
     staticMethods:
-      get_schema_version: '(group: Group) -> uint64_t'
-      set_schema_version: '(group: Group, version: uint64_t)'
+      get_schema_version: '(group: Group) -> SchemaVersion'
+      set_schema_version: '(group: Group, version: SchemaVersion)'
```

Comment:
Not specific to this line, but rather [line 696](https://github.com/realm/realm-js/blob/b3ce947bea5d4b4f29270b09c2b14add85fbe6cf/packages/bindgen/spec.yml#L696) which is commented out.

Could be good to update those as well right?
From (A):
```yml
# apply_schema_changes: '(group: Transaction&, uint64_t: schema_version, target_schema: Schema&, target_schema_version: uint64_t, mode: SchemaMode, changes: std::vector<SchemaChange>, migration_function: () -> void)'
```
To (B):
```yml
# apply_schema_changes: '(group: Transaction&, schema_version: SchemaVersion, target_schema: Schema&, target_schema_version: SchemaVersion, mode: SchemaMode, changes: std::vector<SchemaChange>, migration_function: () -> void)'
```

Note: Also fixed the structure of the 2nd param in (A).

Reply:
done. Although we will likely end up just deleting the whole `ObjectStore` class, since I don't think it is used at all.